javascript-view-swapping/main.js

```diff
@@ -0,0 +1,24 @@
+var $tabContainer = document.querySelector('.tab-container');
+var $tab = document.querySelectorAll('.tab');
+var $view = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', function (event) {
+  if (event.target.matches('div.tab')) {
+    for (var i = 0; i < $tab.length; i++) {
+      if ($tab[i] === event.target) {
+        $tab[i].className = 'tab active';
+      } else {
+        $tab[i].className = 'tab';
+      }
+    }
+  }
+  var $dataView = event.target.getAttribute('data-view');
+  for (var x = 0; x < $view.length; x++) {
+    if ((event.target.matches('div.tab') && $dataView) === ($view[x].getAttribute('data-view'))) {
```

Comment:
Your parentheses are messing up your condition here, you can remove them

Reply:
<img width="912" alt="Screen Shot 2022-07-14 at 4 54 29 PM" src="https://user-images.githubusercontent.com/97363006/179120137-a86bee40-38cc-4320-9d07-e4c97dac0ac6.png">